### PR TITLE
Add -P option to prompt for password.

### PR DESCRIPTION
**Pull request by MagerValp**

### CODE REVIEW

**Comment by gregneagle** [comment]
Looks good to me.

File: createuserpkg [hunk 1]
```diff
@@ -50,12 +53,17 @@ def main():
...
     missing_required_options = False
     for option in required_options:
         if not hasattr(options, option) or getattr(options, option) == None:
             print >> sys.stderr, 'Missing required option: %s' % option
             missing_required_options = True
+    haspassword = getattr(options, 'password') is not None
+    hasprompt = options.prompt == True
+    if haspassword == hasprompt:
+        print >> sys.stderr, 'Must provide password or prompt option'
+        missing_required_options = True
```
Comment (gregneagle) [inline createuserpkg:66]: I wonder, though, if it might just be more friendly to prompt if -p isn't given...

Comment (gregneagle) [reply createuserpkg:66]: IOW, use the password passed in via -p, else prompt. We could then either leave -P in or not, it really wouldn't matter.

Comment (MagerValp) [reply createuserpkg:66]: Yep, that works too, your call.

Comment (gregneagle) [reply createuserpkg:66]: Removing -P for this would free that up for "picture", should someone want to add that...

Comment (gregneagle) [reply createuserpkg:66]: Done in 544c5d0

**Pull request merged**